src/main/java/org/usfirst/frc/team1758/robot/commands/StrafeForward.java

```diff
@@ -26,7 +26,7 @@ protected void execute() {
 			finished = true;
 			driveTrain.mecanumDriveCartesian(0, 0, 0, 0);
 		} else {
-			driveTrain.mecanumDriveCartesian(-.3, 0, 0, sensors.getGyroAngle());
+			driveTrain.mecanumDriveCartesian(0, -.3, 0, sensors.getGyroAngle());
```

Comment:
Should this still be called Strafe Forward now that it is just moving forward now?